package.json

```
@@ -1,20 +1,27 @@
 {
-  "source": "src/index.pug",
+  "source": "src/index.html",
+  "type": "module",
   "scripts": {
-    "watch": "parcel ",
-    "build": "parcel build src/index.pug src/pages/chat/chat.pug src/pages/500/500.pug src/pages/404/404.pug",
-    "start": "node server.js"
+    "watch": "parcel  --no-cache",
+    "build": "parcel build src/index.html src/pages/404/404.html src/pages/500/500.html src/pages/profile/password-edit/password-edit.html src/pages/profile/profile-edit/profile-edit.html src/pages/profile/profile-view/profile-view.html src/pages/auth/login/login.html src/pages/auth/registration/registration.html src/pages/chat/chat.html",
```

Можно использовать "wildcards". Так должно работать:
```suggestion
    "build": "parcel build src/index.html src/pages/**/*.html",
```